### PR TITLE
add cfg-cli package

### PR DESCRIPTION
**Pull request by alexbode**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/cfg-cli/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 0786048d63e5b66338307fb9739142aa06a090744e00141c95c0b8054b4c767b
+
+build:
+  number: 0
+  entry_points:
+    - prism = cfg_cli.prism_cli:wrapper
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (scopatz) [inline recipes/cfg-cli/meta.yaml:16]: This can be `noarch: python`

File: recipes/cfg-cli/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    - prism --help
+
+about:
+  home: "https://github.com/CofactorGenomics/immunoprism-cli"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: "Command-line tool for Cofactor Genomics' products and services."
+  doc_url: "https://cofactorgenomics.github.io"
+  dev_url:
```
Comment (scopatz) [inline recipes/cfg-cli/meta.yaml:48]: Please remove empty entries

File: recipes/cfg-cli/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: "Command-line tool for Cofactor Genomics' products and services."
+  doc_url: "https://cofactorgenomics.github.io"
+
+extra:
+  recipe-maintainers:
+    - alexbode
+    - davemessina
```
Comment (scopatz) [inline recipes/cfg-cli/meta.yaml:53]: @davemessina are you OK with being a maintainer?

Comment (DaveMessina) [reply recipes/cfg-cli/meta.yaml:53]: @scopatz yep, thank you!

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**